<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="inputText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="codedText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="codeButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Закодировать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="blockSize">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
